Basic/2DSpaceShooter/ProjectSettings/Packages/com.unity.learn.iet-framework/Settings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "m_Dictionary": {
+        "m_DictionaryValues": [
+            {
+                "type": "System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089",
+                "key": "IET.DisplayWelcomeDialogOnStartup",
+                "value": "{\"m_Value\":false}"
```

Comment:
this needs to be true, otherwise the welcome dialog will not show up. 

P.S: Be careful about pushing changes to this file, because the flag gets reset to false when you open the project for the first time.

Reply:
it resets every time you close the popup